packages/forma-36-react-components/tools/plop-templates/components/component.tsx.hbs

```diff
@@ -22,19 +22,16 @@ const defaultProps: Partial<{{pascalCase name}}Props> = {
   testId: 'cf-ui-{{kebabCase name}}',
 };
 
-export class {{pascalCase name}} extends Component<{{pascalCase name}}Props> {
-  static defaultProps = defaultProps;
+export function {{pascalCase name}}({className, children, testId, ...otherProps}: {{pascalCase name}}Props) {
```

Comment:
Nice!
Since you are changing the template to use functional components, could you change here how we pass the default props?

so instead of having them in a const in line 21 and doing `{{pascalCase name}}.defaultProps = defaultProps;` in the end

we just need to give `testId` a value in the arg of the function:
```js
function {{pascalCase name}}({className, children, testId = 'cf-ui-{{kebabCase name}}', ...otherProps}
```